packages/eslint-plugin/src/rules/prefer-standalone-component.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  ASTUtils,
+  RuleFixes,
+  isNotNullOrUndefined,
+  Selectors,
+} from '@angular-eslint/utils';
+import type { TSESTree } from '@typescript-eslint/utils';
+import { createESLintRule } from '../utils/create-eslint-rule';
+
+type Options = [];
+export type MessageIds = 'preferStandaloneComponent' | 'suggestAddStandalone';
+export const RULE_NAME = 'prefer-standalone-component';
+const METADATA_PROPERTY_NAME = 'standalone';
+const IS_STANDALONE = 'true';
+
+export default createESLintRule<Options, MessageIds>({
+  name: RULE_NAME,
+  meta: {
+    type: 'suggestion',
```

Comment:
I don't think this should be a suggestion, it should just be an autofix.

Suggestions are intended for cases where there is potentially ambiguity around the resolution to an issue, and the user needs to factor in the context of their specific code - the lint rule can't know in 100% of cases what the right thing to do is.

Here, however, the resolution is very clear and unambiguous - the standalone property needs to be added.

If you want to have a mixture of standalone and non-standalone components you simply wouldn't want to use this rule.

Reply:
I have changed the meta type to `problem` instead of `suggestion` to avoid ambiguous rule usage (I could not find the `autofix` you are mentioning, is it something I need to set in the rule meta?)

Reply:
I’m sorry for the confusion - suggestion is actually correct here. 

This comment should have been against the hasSuggestions: true bit - it should instead use fixable: ‘code’ to enable auto fixing

sorry on mobile so not easy to format the updates there 

please revert back to suggestion as the meta type and sorry again for the confusion

Reply:
No problem, I have reverted the type to `suggestion` and changed the `hasSuggestions: true` to `fixable: code` instead. I'm relatively new to the eslint rules, apologies for the confusion 😄